test/integration/multinode_test.go

```diff
@@ -549,6 +550,9 @@ func validatePodsPingHost(ctx context.Context, t *testing.T, profile string) {
 			continue
 		}
 		hostIP := net.ParseIP(strings.TrimSpace(out.Stdout.String()))
+		if hostIP == nil {
+			t.Errorf("minikube host ip is nil: %s", out.Output())
```

Comment:
```suggestion
			t.Fatalf("minikube host ip is nil: %s", out.Output())
```
nit: This can be a `Fatal` because if this `hostIP` is nil the ping below is guaranteed to fail